src/appdev/Authenticate.ts

```diff
@@ -0,0 +1,62 @@
+import { NextFunction, Response, Request } from 'express';
+import UserSessionRepo from '../repos/UserSessionRepo';
+import { AppDevResponse } from './AppDevResponse';
+
```

Comment:
Delete extra line

Reply:
I put the extra line here to separate imports from a function declaration